### PR TITLE
New package: STM32CubeMX-5.6.1.

### PR DESCRIPTION
**Pull request by ericonr**
I have to check the licensing regarding redistribution, if anyone could help me that would be awesome. https://www.st.com/content/ccc/resource/legal/legal_agreement/license_agreement/group0/39/50/32/6c/e0/a8/45/2d/DM00218346/files/DM00218346.pdf/jcr:content/translations/en.DM00218346.pdf

It should be cross platform, but I haven't managed to take a look at musl yet, and I don't think my RPi can take it. If anyone could give it a spin that would be great too!

### CODE REVIEW

**Comment by ericonr** [comment]
Confirmed to work on musl. Since it's only java, I think it is fully cross platform indeed. Still have to check if it's restricted.

**Comment by ericonr** [comment]
Gonna try to avoid vendoring in the license

**Comment by ericonr** [comment]
fixed c:

File: srcpkgs/STM32CubeMX/template [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+repository=nonfree
+nostrip=yes
+nopie=yes
+
+do_build() {
+	convert "help/STM32CubeMX.ico[17]" stm32cubemx.png
+
+	rm -rf intInstall/
+
+	pdftotext -l 2 -eol unix -nopgbrk help/software_license_agreement.pdf LICENSE
```
Comment (Piraty) [inline srcpkgs/STM32CubeMX/template:23]: why not just vlicense the pdf ?

Comment (ericonr) [reply srcpkgs/STM32CubeMX/template:23]: iirc, @abenson recommended it.

**Pull request closed without merging**